rero_ils/modules/documents/utils.py

```diff
@@ -507,14 +507,58 @@ def create_contributions(contributions):
 
 def get_remote_cover(isbn):
     """Document cover service."""
-    cover_service = current_app.config.get('RERO_ILS_THUMBNAIL_SERVICE_URL')
-    url = cover_service + '?height=60px&jsonpCallbackParam=callback'\
-                          '&type=isbn&width=60px&callback=thumb&value=' + isbn
-    response = requests.get(
-        url, headers={'referer': flask_request.host_url})
+    if isbn:
+        cover_service = current_app.config.get(
+            'RERO_ILS_THUMBNAIL_SERVICE_URL'
+        )
+        url = '{cover_service}' \
+            '?height=244px' \
+            '&width=244px' \
+            '&jsonpCallbackParam=callback' \
+            '&callback=thumb' \
+            '&type=isbn' \
+            '&value={isbn}'.format(
+                cover_service=cover_service,
+                isbn=isbn
+            )
+        try:
+            host_url = flask_request.host_url
+        except:
+            host_url = current_app.config.get('RERO_ILS_APP_URL', '??')
+            if host_url[-1] != '/':
+                host_url = '{host_url}/'.format(host_url=host_url)
```

Comment:
https://www.python.org/dev/peps/pep-0498/#:~:text=F%2Dstrings%20provide%20a%20way,which%20contains%20expressions%20inside%20braces.

With python 3.6, we can skip the ".format" and use simply `f'{host_url}/'`